modules/ding_tabroll/ding_tabroll.features.inc

```diff
@@ -29,8 +29,26 @@ function ding_tabroll_image_default_styles() {
   // Exported image style: ding_tabroll_large.
   $styles['ding_tabroll_large'] = array(
     'name' => 'ding_tabroll_large',
+    'label' => 'ding_tabroll_large',
```

Comment:
If ding_tabroll has defined a imagecache preset which uses manualcrop then manualcrop should also be a dependency for the module size generation of derived images will fail if manualcrop is not enabled.